velox/connectors/hive/HiveConnectorUtil.cpp

```diff
@@ -634,12 +636,16 @@ namespace {
 bool applyPartitionFilter(
     const TypePtr& type,
     const std::string& partitionValue,
-    common::Filter* filter) {
+    common::Filter* filter,
+    const SplitReader::TableFormat& tableFormat) {
```

Comment:
Its better to move the read-only const& parameters before the writable * ones. So lets move tableFormat to the first parameter.